### ISSUE TITLE
JW player won't be fullscreen anymore

### ISSUE THREAD

**Issue by mehrdaddolatkhah** [jwplayer/jwplayer-sdk-android-demo#71]
Hey guys,
I used JW player for my app. previously it's work fine with full screen but now it won't work anymore for fullscreen mode. may please help me to fix this

**Comment by Nilzor** [comment]
Same here. Getting these two log lines in chromium:

```
2019-01-07 12:02:37.013 20198-20198/com.jwplayer.opensourcedemo I/chromium: [INFO:CONSOLE(10)] "Uncaught (in promise) TypeError: fullscreen error", source: http://intercept.jw//data/user/0/com.jwplayer.opensourcedemo/files/jwplayer/jw_core/jwplayer.core.controls.js (10)
2019-01-07 12:02:37.446 20198-20198/com.jwplayer.opensourcedemo I/chromium: [INFO:CONSOLE(10)] "Uncaught (in promise) TypeError: Document not active", source: http://intercept.jw//data/user/0/com.jwplayer.opensourcedemo/files/jwplayer/jw_core/jwplayer.core.controls.js (10)
```

Also tried overriding the fullscreen handler by adding the following code on line 56 in `JWPlayerViewExample.java`. Result: `onFullscreenRequested` is never invoked - logcat does  not output "FULLSCREEN-REQ":

```
        mPlayerView.setFullscreenHandler(new FullscreenHandler() {
            @Override
            public void onFullscreenRequested() {
                android.util.Log.d("TAG","FULLSCREEN-REQ");
            }
            // the rest of the methods are empty overrides
        });
```

Tested on emulator and Samsung S6. Current master branch of repo, meaning JWPlayer version `3.2.2`

**Comment by Nilzor** [comment]
Ping . This is really a blocking issue for any use of your SDK

**Comment by okulagin** [comment]
@Nilzor the fix was released in 3.4.1. Thank you.

**Comment by Nilzor** [comment]
Thanks :+1: verified working!
